### DISCUSSION TITLE
my groundtruth and prdict_mask always black

### DISCUSSION THREAD

**Discussion by Phyonna** [open-edge-platform/anomalib#1837] [Q&A]
I'm trying to train the model on my own dataset. As you can see, my grand truth has the white mask.But after training, the result shows that my ground truth and predict masks are all black. Could you please help me? 
![image](https://github.com/openvinotoolkit/anomalib/assets/102140144/eb048f56-ba91-4651-bfd9-cec7806968a7)
![image](https://github.com/openvinotoolkit/anomalib/assets/102140144/8c07dbd5-a942-4778-a6e4-2d8e7109a341)

**Comment by blaz-r** [comment]
This can happen with adaptive threshold and if the model is not very certain in the prediction. You could try manual threshold, or improve the overall model performance.
